src/components/AddPlaidBankAccount.js

```diff
@@ -112,6 +113,7 @@ class AddPlaidBankAccount extends React.Component {
             label: `${account.addressName} ${account.mask}`,
         }));
         const {icon, iconSize} = getBankIcon();
+        const plaidDataErrorMessage = this.props.plaidData.errors ? _.chain(this.props.plaidData.errors).values().first().value() : this.props.plaidData.error;
```

Comment:
Thanks for moving this to a variable.

I think there will be a problem when `this.props.plaidData.errors = {}` because that's considered "truthy" and so the message will be set to the undefined first value. You could fix that with `_.isEmpty(this.props.plaidData.errors)`.

I guess it's fine to use underscore.js here to get the first error value.

Reply:
smart yeah, I just checked it and you're right. I thought that since empty strings evaluate to false, empty objects do as well but was wrong.

Reply:
Ah yes the horrors of javascript 😂